usaspending_api/search/v2/views/search.py

```diff
@@ -885,9 +921,9 @@ class SpendingByTransactionCountVisualizaitonViewSet(APIView):
     @cache_response()
     def post(self, request):
 
-        models = [{'name': 'keyword', 'key': 'filters|keyword', 'type': 'text', 'text_type': 'search', 'min': 3}]
+        models = [{'name': 'keyword', 'key': 'filters|keyword', 'type': 'array', 'array_type': 'text',
+                  'text_type': 'search', 'optional': False}]
```

Comment:
lost the `min` value of 3. That is a desired constraint

Reply:
Good catch here, this should have been separated out to the other ticket.